packages/eas-cli/src/build/ios/build.ts

```diff
@@ -45,6 +45,14 @@ export async function prepareIosBuildAsync(
       buildCtx.commandCtx.projectDir,
       iosBuildScheme
     );
+  } else {
+    if (!commandCtx.exp.name) {
+      throw new Error('"expo.name" is required in your app.json');
+    }
```

Comment:
Is this check necessary? I think it is always defined.

Reply:
I prefer to check that just in case, e.g. someone can have `name: ''` in app.json